siliconcompiler/targets/asap7_demo.py

```diff
@@ -21,7 +21,7 @@ def setup(chip):
     from flows import asicflow
     from libs import asap7sc7p5t
     chip.use(asap7)
-    chip.use(asicflow)
+    chip.use(asicflow, syn_np=syn_np, floorplan_np=floorplan_np, physyn_np=physyn_np, place_np=place_np, cts_np=cts_np, route_np=route_np, dfm_np=dfm_np)
```

Comment:
With such a large number of arguments, clarity argument goes away. For something like this, a dictionary seems more natural. Something to think about...

Reply:
I can convert this to be a dict that is expanded and passed in so it's a much shorter line to read.

Reply:
I was considering proposing a clever solution to this, but I'm not sure if it would be worth it. You could probably use `inspect.signature`:

```
import inspect

def setup(chip, syn_np=1, floorplan_np=1, physyn_np=1, place_np=1, cts_np=1, route_np=1, dfm_np=1):
    for arg in inspect.signature(setup).parameters:
        if arg != 'chip':
            kwargs[arg] = locals()[arg]
    [...]
    chip.use(asicflow, **kwargs)
```

But again, I think it's probably too complex to save a very small amount of repetition.

Reply:
I think we can explore that in a separate PR if it's worth it, but I would worry that users looking at these modules will not be able to follow easily.

Reply:
Yeah, and it would be hard to debug later.